public/Highways.php

```diff
@@ -28,6 +28,7 @@ function index() {
    * @param int $id The database ID of the highway you'd like to view.
    * @return [Highway] The highway requested.
    */
+  //Do NOT change the return statement to @return Highway [Highway] unless you want a bad day
```

Comment:
PHPStorm was complaining that the PHPDoc wasn't formatted correctly so I let it auto generate the correct formatting then promptly forgot about it and went on to do other things. 

Turns out that change breaks literally every api endpoint and I pulled my hair out for an hour trying to figure out why. 

Another reason why comments dictating code is sad day.